core/wiki/macros/toc.tid

```diff
@@ -2,182 +2,212 @@ title: $:/core/macros/toc
 tags: $:/tags/Macro
 
 \define toc-caption()
-<$set name="tv-wikilinks" value="no">
-  <$transclude field="caption">
-    <$view field="title"/>
-  </$transclude>
-</$set>
+\whitespace trim
+<$set name="tv-wikilinks" value=no>
```

Comment:
The coding style for the core is to default to use double quotes around attribute values.